package.json

```diff
@@ -20,7 +20,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@gomomento/sdk": "^1.77.0",
+    "@gomomento/sdk": "1.80.0",
```

Comment:
This is the thing I was showing in our PR review meeting last Wednesday. Because we are exposing CacheClient and related classes as part of the API for this library, if you pin to a specific version, you are increasing the chances that they will end up with two conflicting versions of the dependency and run into those weird class compatibility errors.

Reply:
Ah, true. Will fix it